core/sqf/monitor/linux/cluster.h

```diff
@@ -229,6 +235,7 @@ class CCluster
     int     configPNodesMax_;   // max # of physical nodes that can be configured
     int    *NodeMap;        // Mapping of Node ranks to COMM_WORLD ranks
     int     TmLeaderNid;    // Nid of currently assigned TM Leader node
+    int     MonitorLeaderPNid; // PNid of currently assigned Monitor leader node
```

Comment:
In much of the SQL code, the convention is that member names end in an underscore. Do we follow that convention in the monitor?

Reply:
Yes, we do. Noted these anomalies in the code this morning and will change this to the followed convention. Thanks for pointing this out.